src/core/lib/experiments/rollouts.yaml

```diff
@@ -100,3 +100,5 @@
   default: true
 - name: combiner_offload_to_event_engine
   default: true
+- name: rm_lookup_point_transport
```

Comment:
Suggest calling this `registered_method_lookup_in_transport`.

Reply:
Done